dbt/include/spark/macros/materializations/incremental.sql

```diff
@@ -60,21 +60,24 @@
 
 {% endmacro %}
 
-{% macro get_merge_sql(source, target, unique_key) %}
+
+{% macro spark__get_merge_sql(target, source, unique_key, dest_columns, predicates=[]) %}
```

Comment:
A [bird](https://github.com/drewbanin) once told me that setting a default argument of `[]` is dangerous (for reasons that have do with [this](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)), and that it's preferable to set the default to `none` and include a statement like [this](https://github.com/fishtown-analytics/dbt/blob/dev/octavius-catto/core/dbt/include/global_project/macros/materializations/common/merge.sql#L19).

Reply:
We never actually use the argument, but I am happy to just change it to `none`!